config/subsidy-application-management/versions/20210208105912-remove-aanvraagdatum-from-application-form/config.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
@claire-lovisa why do we create a completely new form version?

1. Nothing breaking has changed to the form spec.
2. The newest form version, being the one for "contact-tracing" hasn't even been released on test and prod. yet

Reply:
@HugaertsDries wanted to keep it clean but that's indeed a bit overkill, really small change with ... almost no impact. And good remark about the contact tracing one, will merge it there :)